SplitIt/app/src/main/java/project/cs495/splitit/ReceiptViewActivity.java

```diff
@@ -361,12 +387,44 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
 
-        if (id == R.id.add_item) {
-            addItem();
+        if (id == R.id.create_user_receipt) {
+            createUserReceipt();
         }
         return super.onOptionsItemSelected(item);
     }
 
+    private void createUserReceipt() {
+        final String userId = FirebaseAuth.getInstance().getCurrentUser().getUid();
+        UserReceiptBuilder userReceiptBuilder = (UserReceiptBuilder) new UserReceiptBuilder()
+                .setReceiptId(receiptId)
+                .setCreator(receipt.getCreator())
+                .setDatePurchased(receipt.getDatePurchased())
+                .setPrice(receipt.getPrice())
```

Comment:
This is the total price.  Need to add items up and set the sum of the items.  Once fixed this PR is gtg.